package.json

```diff
@@ -75,6 +76,7 @@
     "@sentry/node": "^5.7.1",
     "@uphold/uphold-sdk-javascript": "github:brave-intl/uphold-sdk-javascript#bump-lodash-merge",
     "accept-language-parser": "1.4.1",
+    "avro-js": "1.9.1",
```

Comment:
tried npm i locally and did not have any luck. is there a specific node version that should be used i tried 8.9.2 and 10.15.3

Reply:
whaaa? there should not be any build issues with this, it's native javascript. did you have problems with `sinek` maybe?

Reply:
sorry. yeah. could be. just npm i failed. not sure which part